components/listingsPage/filters.js

```diff
@@ -237,43 +280,113 @@ const Filters = ({ t }) => {
-                onChange={(e) => handleInsuranceType(e)}
-                label={type}
-                name="trip_type"
-                value={value}
-              />
-            )
-          )}
-        </FilterFormFIeld>
-        {/* <FilterFormFIeld>
-          <Legend>
-            {t("headings.filter_by_length_of_trip")}
-            <Tooltip hover text="To be supplied" />
-          </Legend>
-          {t("filters.tripLength", { returnObjects: true }).map(
-            ({ length, value }, i) => (
-              <Radio
-                key={i}
-                checked={tripLength.tripLength === value}
-                onChange={(e) => handleTripLength(e)}
-                label={length}
-                name="tripLength"
-                id={`length-${i}`}
-                value={value}
-              />
-            )
-          )}
-        </FilterFormFIeld> */}
+        {/* Client Insurance Type */}
+        {process.browser ? (
+          <FilterFormFIeld>
+            <Legend>
+              {t("headings.filter_by_insurance_type")}
+              <Tooltip minWidth="300px" hover text={insuranceTypeTip} />
+            </Legend>
+            {t("filters.trip_type", { returnObjects: true }).map(
+              ({ type, value }, i) => (
+                <Radio
+                  key={i}
+                  checked={trip_type.trip_type === value}
+                  onChange={(e) => handleInsuranceType(e)}
+                  label={type}
+                  name="trip_type"
+                  value={value}
+                />
+              )
+            )}
+            <Heading level={4}>
+              {t("headings.filter_by_length_of_trip")}
+            </Heading>
+            <Note note={note}>Please select the type of insurance first</Note>
+            <SingleTrip single={single}>
+              {t("filters.singleTripLength", { returnObjects: true }).map(
+                ({ length, value }, i) => (
+                  <Radio
+                    key={i}
+                    checked={singleOption.singleOption === value}
+                    onChange={(e) => handleSingleOption(e)}
+                    label={length}
+                    name="singleOption"
+                    value={value}
+                  />
+                )
+              )}
+            </SingleTrip>
+            <AnnualTrip annual={annual}>
+              <span style={{ fontSize: "14px" }}>for each individual trip</span>
+
+              {t("filters.annualTripLength", { returnObjects: true }).map(
+                ({ length, value }, i) => (
+                  <Radio
+                    key={i}
+                    checked={annualOption.annualOption === value}
+                    onChange={(e) => handleAnnualOption(e)}
+                    label={length}
+                    name="annualOption"
+                    value={value}
+                  />
+                )
+              )}
+            </AnnualTrip>
```

Comment:
In my opinion both the `SingleTrip` and `AnnualTrip` should be inside another `FieldSet` using `legend` instead of the `Heading` as well